torch/distributions/bijectors.py

```diff
@@ -0,0 +1,79 @@
+from torch.distributions.utils import broadcast_all
+
+
+class Bijector:
+    """
+    Abstract class `Bijector`. `Bijector` are bijective transformations with computable
+    inverse log det jacobians. They are meant for use in `TransformedDistribution`.
+    """
+    is_injective = True
+    add_inverse_to_cache = False
+
+    def __init__(self):
+        self._intermediates_cache = {}
+
+    def __call__(self, base_distribution):

```

Comment:
This method seems surprising to me. I think it's clearer to omit this method and simply call a `TransformedDistribution(...)`. WDYT?